packages/react-select/src/components/Menu.js

```diff
@@ -383,9 +386,13 @@ export const MenuList = (props: MenuListComponentProps) => {
     innerProps,
     innerRef,
     isMulti,
+    selectProps:{ menuIsOpen }
   } = props;
   return (
-    <div
+    <ul
```

Comment:
I don't think we should change the tags to `<ul>` and `<li>`. They are no requirement for a list of selectable items. `role` should be sufficient enough for this.